eng/Versions.props

```diff
@@ -13,6 +13,7 @@
     </MicrosoftNETSdkInternalPackageVersion>
     <MicrosoftWindowsDesktopAppRuntimewinx64PackageVersion>5.0.4</MicrosoftWindowsDesktopAppRuntimewinx64PackageVersion>
     <MicrosoftAspNetCoreAppRuntimewinx64PackageVersion>5.0.4</MicrosoftAspNetCoreAppRuntimewinx64PackageVersion>
+    <MicrosoftNetILLinkTasksVersion>7.0.100-1.22057.1</MicrosoftNetILLinkTasksVersion>
```

Comment:
Did you also use darc to add this dependency to our subscription? That way we can ensure that this value is automatically updated going forward.

Reply:
I did not yet, I'll add that